quantulum3/_lang/en_US/regex.py

```diff
@@ -62,7 +62,7 @@
 
 MULTIPLICATION_OPERATORS = {" times "}
 
-DIVISION_OPERATORS = {" per ", " a "}
+DIVISION_OPERATORS = {" per ", " a ", " / "}
```

Comment:
Is there any test case that covers the addition of " / " as a division operator?

Reply:
As this PR is about numbers and not units (division operators are taking care of parsing "mm per inch" or similar units), I will take this change out of this PR